component-library/components/generic/component-heading/component-heading.scss

```diff
@@ -0,0 +1,73 @@
+.c-component-heading {
+  color: var(--brand-grey-000, #f9f9fb);
+  background: var(--brand-grey-900, #1b1b1d);
+  display: grid;
+  grid-template-columns: var(--twelve-column-grid);
```

Comment:
I felt so as grid stuff should inherit from parent component. Because component heading is a generic component, it shouldn't worry about its position in the whole layout, but should only worry about its position within the component.